src/test/java/net/snowflake/client/jdbc/PreparedStatement1IT.java

```diff
@@ -805,16 +776,17 @@ public void manualTestForPreparedStatementLogging() throws SQLException {
     props.put("user", params.get("user"));
     props.put("password", params.get("password"));
     props.put("tracing", "info");
-    Connection con = DriverManager.getConnection(uri, props);
-    con.createStatement()
-        .executeUpdate("alter session set CLIENT_ENABLE_LOG_INFO_STATEMENT_PARAMETERS=true");
-    con.createStatement().execute(createTableSQL);
-    PreparedStatement prepStatement = con.prepareStatement(insertSQL, Statement.NO_GENERATED_KEYS);
-    bindOneParamSet(prepStatement, 1, 1.22222, (float) 1.2, "test", 12121212121L, (short) 12);
-    prepStatement.addBatch();
-    prepStatement.executeBatch();
-    con.createStatement()
-        .executeUpdate("alter session set CLIENT_ENABLE_LOG_INFO_STATEMENT_PARAMETERS=false");
-    con.close();
+    try (Connection con = DriverManager.getConnection(uri, props);
+        Statement statement = con.createStatement()) {
+      statement.executeUpdate("alter session set CLIENT_ENABLE_LOG_INFO_STATEMENT_PARAMETERS=true");
+      statement.execute(createTableSQL);
+      PreparedStatement prepStatement =
```

Comment:
this prepared statement should be also in try with resources?

Reply:
I wrapped up all the PreparedStatement with try with resources in this file.